citystatitemdelegate.h

```diff
@@ -0,0 +1,23 @@
+#ifndef CITYSTATITEMDELEGATE_H
+#define CITYSTATITEMDELEGATE_H
+
+#include <QItemDelegate>
+
+
+class CityStatItemDelegate : public QItemDelegate
+{
+    Q_OBJECT
+public:
+    CityStatItemDelegate(QObject *parent = 0);
+
+    QWidget *createEditor(QWidget *parent, const QStyleOptionViewItem &option, const QModelIndex &index) const;
```

Comment:
All overridden methods should be marked virtual and put into private section.

Reply:
done